news/3913.doc.rst

```diff
@@ -0,0 +1 @@
+Define python_version value.
```

Comment:
```suggestion
Clarify the proper value of `python_version`.
```

Reply:
One more variable has been documented, can't accept the edit, ignoring.